core/kazoo_services/src/kazoo_services_maintenance.erl

```diff
@@ -149,7 +149,11 @@ js_quantify_map() ->
        "        case 'user':"
        "            emit(['users', doc.priv_level], 1);"
        "            if (doc.qubicle && doc.qubicle.enabled) {"
-       "                emit(['qubicle', 'recipients'], 1);"
+       "                if (doc.qubicle.recipient && doc.qubicle.recipient.offering == 'pro') {"
+       "                    emit(['qubicle', 'pro_recipient'], 1);"
+       "                } else {"
```

Comment:
@mark2600 while there is only two offerings currently, why not just make this generic like the Erlang version (they should be functional equivalents)?

if (doc.qubicle.recipient) {
    emit(['qubicle', doc.qubicle.recipient.offering || 'basic' + '_recipient'], 1);"
}